src/main/scala/cromwell/util/google/GoogleCredentialFactory.scala

```diff
@@ -114,4 +116,14 @@ abstract class GoogleCredentialFactory {
       case unrecognized => Failure(new IllegalArgumentException(s"Unrecognized userSchemeAuthentication: $unrecognized"))
     } getOrElse Failure(new Throwable("No user authentication configuration has been found."))
   }
+
+  private def forApplicationDefaultCredentials(): Credential = {
+    try {
+      GoogleCredential.getApplicationDefault().createScoped(GoogleScopes.Scopes.asJava)
+    } catch {
+      case e : IOException =>
+      log.warn("Failed to get application default credentials", e)
+      throw e
```

Comment:
OK, it's possible the caller wasn't logging the exception.  Unfortunately I don't think we have a policy on how this ought to be done in Cromwell.